src/utils/crypto-utils.ts

```diff
@@ -0,0 +1,20 @@
+import { createDecipheriv } from "crypto"
+
+import { config } from "@config/config"
+
+const ALGORITHM = "aes-256-cbc"
+
+export const decryptPassword = (encryptedPassword: string, iv: string) => {
```

Comment:
**highly** suggest typing + branding the return and enforcing this on downstream callers. doesn't have to be here if the logic here is correct.

```ts
// type-def of decrypted password, prevents assigning string to it
type DecryptedPassword = Brand<string, "DecryptedPassword">

export const decryptPassword = (): DecryptedPassword
```

Reply:
Updated in https://github.com/isomerpages/isomercms-backend/pull/806/commits/cea7e0e6abd457d5305c03b41508533f3c30d815!